src/index.js

```diff
@@ -31,22 +31,22 @@ import UserChip from './UserChip/UserChip.jsx';
 import CircleMenu from './CircleMenu/CircleMenu.jsx';
 import Window from './Window/Window.jsx';
 
-import CapabilitiesUtil from './Util/CapabilitiesUtil/CapabilitiesUtil';
-import CsrfUtil from './Util/CsrfUtil/CsrfUtil';
-import AnimateUtil from './Util/AnimateUtil/AnimateUtil';
-import FeatureUtil from './Util/FeatureUtil/FeatureUtil';
-import FileUtil from './Util/FileUtil/FileUtil';
-import GeometryUtil from './Util/GeometryUtil/GeometryUtil';
-import MapUtil from './Util/MapUtil/MapUtil';
-import MathUtil from './Util/MathUtil/MathUtil';
-import MeasureUtil from './Util/MeasureUtil/MeasureUtil';
-import ObjectUtil from './Util/ObjectUtil/ObjectUtil';
-import ProjectionUtil from './Util/ProjectionUtil/ProjectionUtil';
-import StringUtil from './Util/StringUtil/StringUtil';
-import UndoUtil from './Util/UndoUtil/UndoUtil';
-import UrlUtil from './Util/UrlUtil/UrlUtil';
-import WfsFilterUtil from './Util/WfsFilterUtil/WfsFilterUtil';
-import Logger from './Util/Logger';
+import { CapabilitiesUtil } from '@terrestris/ol-util';
+import { CsrfUtil } from '@terrestris/base-util';
+import { AnimateUtil } from '@terrestris/ol-util';
+import { FeatureUtil } from '@terrestris/ol-util';
+import { FileUtil } from '@terrestris/ol-util';
+import { GeometryUtil } from '@terrestris/ol-util';
+import { MapUtil } from '@terrestris/ol-util';
+import { MathUtil } from '@terrestris/base-util';
+import { MeasureUtil } from '@terrestris/ol-util';
+import { ObjectUtil } from '@terrestris/base-util';
+import { ProjectionUtil } from '@terrestris/ol-util';
+import { StringUtil } from '@terrestris/base-util';
+import { UndoUtil } from '@terrestris/base-util';
+import { UrlUtil } from '@terrestris/base-util';
+import { WfsFilterUtil } from '@terrestris/ol-util';
+import { Logger } from '@terrestris/base-util';
```

Comment:
I think all util imports should be removed from the `index.js` since all classes are imported directly from the corresponding util modules anyway.